src/NHibernate.Test/CompositeId/ClassWithCompositeIdFixture.cs

```diff
@@ -228,5 +228,110 @@ public void Hql()
 				Assert.AreEqual(1, results.Count);
 			}
 		}
+
+		[Test]
+		public void HqlInClause()
+		{
+			//Need to port changes from InLogicOperatorNode.mutateRowValueConstructorSyntaxInInListSyntax
+			if (!Dialect.SupportsRowValueConstructorSyntaxInInList)
```

Comment:
While this functionality is working for dialects where row value constructor is supported - this test is currently ignored for all dialects because `SupportsRowValueConstructorSyntaxInInList` is false for all dialects.  `SupportsRowValueConstructorSyntaxInInList` is set in my other earlier PR #2158

Reply:
Ok